managed_vms/extending_runtime_compat/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# [START dockerfile]
```

Comment:
I was trying to be consistent with the sample I copied this from, but fair point.